src/data/offline.json

```diff
@@ -733,6 +733,28 @@
 		"notes": null,
 		"badge": "OWASP/railsgoat"
 	},
+	{
+		"url": "https://github.com/globocom/secDevLabs",
+		"name": "SecDevLabs",
+		"technology": [
+			"Go",
+			"NodeJS",
+			"Python",
+			"PHP",
+			"React",
+			"Angular/Spring",
+			"Dart/Flutter"
+		],
+		"references": [
+			{
+				"name": "guide",
+				"url": "https://github.com/globocom/secDevLabs"
+			}
+		],
+		"author": "Globo",
+		"notes": "Repository with many intentionally vulnerable web applications. Includes attack naratives and docker options for each app.",

```

Comment:
```suggestion
		"notes": "Repository with many intentionally vulnerable web applications. Includes attack narratives and docker options for each app.",

```

Reply:
Hi, @kingthorin! Thanks for taking the time to review this PR.

I have fixed the typo you pointed out.